bin/_down

```diff
@@ -8,11 +8,11 @@ ROOT=${DIR}/..;
 source ${ROOT}/.env;
 
 if [ -f ${ROOT}/docker-compose.yml ]; then
-    docker-compose -f ${ROOT}/docker-compose.yml down;
+    docker-compose -p ${PROJECT_NAME} -f ${ROOT}/docker-compose.yml down;
 fi;
 
-docker rm $(docker ps -q -f status=exited);
-docker rmi $(docker images -q -f dangling=true);
-docker volume rm $(docker volume ls -qf dangling=true);
+docker rm $(docker ps -q -f status=exited) >/dev/null 2>&1;
+docker rmi $(docker images -q -f dangling=true) >/dev/null 2>&1;
+docker volume rm $(docker volume ls -qf dangling=true) >/dev/null 2>&1;
```

Comment:
Maybe it's a bit dangerous to remove *any* output from these commands?

It might be a good idea to keep at least the error output, don't you think?

Like `>/dev/null` instead of `>/dev/null 2>&1`